Corona-Warn-App/src/main/res/values/strings.xml

```diff
@@ -195,14 +195,14 @@
     ###################################### -->
     <string name="risk_details_headline_behavior">Verhalten Sie sich richtig</string>
     <string name="risk_details_behavior_body_stay_home">Bleiben Sie zu Hause</string>
-    <string name="risk_details_behavior_body_avoid_contact">Vermeiden Sie sozialen Kontakt</string>
+    <string name="risk_details_behavior_body_avoid_contact">Vermeiden Sie soziale Kontakte</string>
     <string name="risk_details_behavior_body_contact_doctor">Für Fragen zu auftretenden Symptomen, Testmöglichkeiten und weiteren Absonderungsmaßnahmen wenden Sie sich bitte an:</string>
     <string name="risk_details_behavior_body_wash_hands">Waschen Sie Ihre Hände</string>
     <string name="risk_details_behavior_body_wear_mask">Tragen Sie eine Schutzmaske</string>
```

Comment:
Perhabs it should be rather "Mund-Nasen-Bedeckung", but these are technical details…